test/unit_tests.proto

```diff
@@ -22,9 +22,20 @@ syntax = "proto3";
 package spine.test;
 
 import "spine/options.proto";
+import "google/protobuf/any.proto";
 
 option (type_url_prefix) = "type.test.spine.io";
 
 message TestMessage {
     string value = 1;
 }
+
+message StuffId {
```

Comment:
Let's have `ProjectId` and `CreateProject` (see below) instead.

And let's avoid `cool_field` and replace it with something more meaningful.